polyglot/piranha/src/config.rs

```diff
@@ -1,103 +0,0 @@
-/*
-Copyright (c) 2022 Uber Technologies, Inc.
-
- <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
- except in compliance with the License. You may obtain a copy of the License at
- <p>http://www.apache.org/licenses/LICENSE-2.0
-
- <p>Unless required by applicable law or agreed to in writing, software distributed under the
- License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
- express or implied. See the License for the specific language governing permissions and
- limitations under the License.
-*/
-
-//! This module contains all the `structs` and implementations required for - (i) handling Piranha's run-time arguments,
-//! (ii) reading language specific configurations, and (iii) API specific configurations.
-//! This module defines all basic building block `structs` used by Piranha.
-
-use crate::{
-  models::piranha_arguments::PiranhaArguments,
-  models::{
-    outgoing_edges::{Edges, OutgoingEdges},
-    rule::{Rule, Rules},
-    scopes::{ScopeConfig, ScopeGenerator},
-  },
-  utilities::{parse_toml, read_toml},
-};
-
-use std::path::Path;
-
-use clap::Parser;
-
-/// A refactoring tool that eliminates dead code related to stale feature flags.
-#[derive(Clone, Parser, Debug)]
-#[clap(name = "Piranha")]
-pub(crate) struct CommandLineArguments {
```

Comment:
This is not required any more. We move command line parsing logic to `PiranhaArguments`